src/Exception/ExceptionInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Bernard\Exception;
+
+/**
+ * Interface for exceptions thrown by Bernard
+ * @package Bernard
+ */
+interface ExceptionInterface
```

Comment:
Would `BernardException` be OK? Saves aliasing plain old `Exception` is my thinking...